src/command_modules/azure-cli-cdn/azure/cli/command_modules/cdn/_help.py

```diff
@@ -0,0 +1,160 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=unused-import
+from azure.cli.core.help_files import helps
+
+# pylint: disable=line-too-long
+helps['cdn'] = """
+    type: group
+    short-summary: Manage Azure Content Delivery Networks (CDN)
+"""
+
+helps['cdn profile create'] = """
+    type: command
+    examples:
+        - name: Create a CDN profile using Verizon premium CDN
+          text: >
+            az cdn profile create -g group -n profile --sku Premium_Verizon
+"""
+
+helps['cdn profile update'] = """
+    type: command
+    short-summary: Updates a CDN profile with a profile name under the specified resource group.
+"""
+
+helps['cdn profile delete'] = """
+    type: command
+    examples:
+        - name: Delete a CDN profile
+          text: >
+            az cdn profile create -g group -n profile
+"""
+
+helps['cdn profile list'] = """
+    type: command
+    short-summary: List your Azure Content Delivery Network (CDN) profiles
+    examples:
+        - name: List CDN profiles
+          text: >
+            az cdn profile list
```

Comment:
There's a tradeoff for examples... if you've run -h on anything with examples, they tend to make your screen scroll.  When the examples help point you to your scenario or guide you to proper usage of parameter "sets" then this tradeoff if worth it (usually with creates or updates).  If there's only one logical way of using the command, then, IMHO, it would be better to omit an example.